src/components/Header/Header.tsx

```diff
@@ -95,7 +96,11 @@ const Header: React.SFC<HeaderProps> = ({
             ) : null}
             <HeaderItem>
               {authStore.isAuth ? (
-                <span className="hidden-xs" onClick={signOut}>
+                <span
+                  id={AnalyticsIds.LogoutButton}
```

Comment:
I don't like an idea that you add id that has reference to analytics. Analytic tool should not force you to change common html. I think the best option here will be data attribute for ids (for all same cases)